azure-mgmt-machinelearningcompute/azure/mgmt/machinelearningcompute/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.2.0"
```

Comment:
So I noticed that before I updated this the AutorestCI changes set this version back to 0.1.0 instead of bumping it. Why does it do that?

Reply:
It's the SwaggerToSdk configuration file here:
https://github.com/Azure/azure-sdk-for-python/blob/3c5985ce0fd86b8b36ecc94a385399c64b113096/swagger_to_sdk_config.json#L339

I didn't find a good way to automatically increment this number yet, since I don't know by advance if the next PR will be breaking or not :(.
Fixing the file manually is then fine.

Reply:
Gotcha, so I should update this file each PR, but like you said it also may not help because we don't know what the next PR will be.